src/Event/PostMappingBuilderEvent.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/*
+ * This file is part of the FOSElasticaBundle package.
+ *
+ * (c) FriendsOfSymfony <https://friendsofsymfony.github.com/>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace FOS\ElasticaBundle\Event;
+
+use FOS\ElasticaBundle\Configuration\IndexConfigInterface;
+use Symfony\Contracts\EventDispatcher\Event;
+
+class PostMappingBuilderEvent extends Event
```

Comment:
Shouldn't this be a transitive verb as other events. Something like `PostMappingBuild`?

Reply:
Good point. Fixed in https://github.com/FriendsOfSymfony/FOSElasticaBundle/pull/1734/commits/3e93c9d9fc41f6f20bd3c1dba50eef49ced6728d